docs/book/releases/v1.0.2.md

```diff
@@ -0,0 +1,61 @@
+<!-- markdownlint-disable MD034 -->
+# vSphere CSI Driver - v1.0.2 release
+
+## New Feature
+
+- Implement the GetVolumeStats CSI node capability ([#108](https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/108), [@flofourcade](https://github.com/flofourcade))
+
+## Other Notable Changes
+
+- Mitigate CVE-2019-11255 by bumping external-provisioner image version to v1.2.2. [CVE announcement](https://groups.google.com/forum/#!topic/kubernetes-security-announce/aXiYN0q4uIw) [#98](https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/98)
+- Remove vsphere credentials from csi node daemonset [#96](https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/96)
+- Support for running the CSI controller outside the controlled cluster. [#102](https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/102)
+- Hotfix compatibility issue on 7.0. Using the correct field name to govmomi. [#127](https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/127)
+- Invoke NodeRegister only when Provider Id change is observed. [#132](https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/132)
+- removing pending status check for the pod Delete [#117](https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/117)
+- Replace container images to match published manifests [#105](https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/105)
+- updating deployment yamls with v1.0.1 images [#86](https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/86)
+- Persistent volume expansion.
```

Comment:
This is offline volume expansion only, right?  Also it is for block only?  We should mention that.

Reply:
clarieid as `Offline Persistent Volume expansion for Block Volume.`.

Reply:
actually this feature was on added in 1.0.2, so remvoed it. there were other corrections in the "Notable Changes" for v1.0.2 release.